### ISSUE TITLE
No sound in Music Files Explorer & Visual Cue Editor and no Artworks in Dashboard Station Overview

### ISSUE THREAD

**Issue by unudelaradio** [AzuraCast/AzuraCast#7648]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

Rolling Release #7138495 (2024-12-30 15:33)

### What happened?

**Issue 1**: Media Files are not playing in Media → Music Files Explorer and Editor. In fact none of the small play buttons in Admin Area are working (i get no sound): including stream previews in Dashboard or previews in Playlist Reorder Popup.
**Issue 2:** Visual Cue Editor also not working. There's no sound when pressing play and no waveform preview generated.

These no sound issues are present only in Admin Area. On Public Pages and Embeded Players everything is fine.

**Issue 3:** In `/dashboard` on Station Overview only the first station in list has visible artwork. Artworks are missing for the rest of the stations in Now Playing column.

**Comment by unudelaradio** [comment]
I also just discovered that reordering tracks in Playlists Reorder Popup doesn't have any effect.

**Comment by LordHelmchen666** [comment]
Where are the files stored? Maybe external?

**Comment by unudelaradio** [comment]
> Where are the files stored? Maybe external?

Files are on local storage. 

I just updated Azuracast on a second instance and i have the same problems. On both instances with the latest Rolling Release something broke. No sound in Dashboard Area and Playlists Reorder function doesn't work anymore. Also Artworks are missing in Dashboard / Station Overview on the second instance too, after the update.

**Comment by LordHelmchen666** [comment]
Seems to me like a permission issue.

**Comment by unudelaradio** [comment]
I think the issues are related to some changes that i just spotted in my dashboard: all streams and public pages are displayed in a new format. There's a port inserted in all the links (6443)

Ex.
mysubdomain.com:6443/listen/station_name/mount_name
mysubdomain.com:6443/public/station_name

Most of the links in my instances now has this new 6443 port, that maybe causes the issues.

**Comment by LordHelmchen666** [comment]
Yes, that could be the reason.
@BusterNeece Could you please Look into this. BTW Wish you a Happy NewYear! Please stay healthy and stay with us for a long time.

**Comment by LordHelmchen666** [comment]
I have updated to Rolling Release #[5482f58](https://github.com/AzuraCast/AzuraCast/commit/5482f58a48a0836fc4f14a9ae102d4145d999a8c) (2024-12-31 12:52). I could not reproduce these errors. Everything is working on my install.

Maybe you need to empty your browser cache? Could you use your browser console https://balsamiq.com/support/faqs/browserconsole/ to see if there is an error?

**Comment by RM-FM** [comment]
Same issue here since I've update to the latest RR [5482f58](https://github.com/AzuraCast/AzuraCast/commit/5482f58a48a0836fc4f14a9ae102d4145d999a8c) today.

**Comment by unudelaradio** [comment]
> EDIT Do you have any special setting? Using docker-compose.override.yml? Using a Proxy? Did you restart docker so changes could take place?

I didn't make any changes to system settings. Just a simple update from an old rolling release (since december 2nd as i remember) to the latest Rolling Release. And after update this 6443 port came out of nowhere in the dashboard links.

**Comment by LordHelmchen666** [comment]
Installed in a Virtualbox VM. There i can see this error.
Error happens with http and https protocols.

![Screenshot 2025-01-01 at 21-13-06 Dashboard - AzuraCast](https://github.com/user-attachments/assets/d560c44a-782c-40ce-84fb-d4823b643697)

Turn on "Prefer Browser URL" then error is gone. So i guess it is something wrong with internal ports now.

![Screenshot 2025-01-01 at 21-13-53 Administration - AzuraCast](https://github.com/user-attachments/assets/be1f7b80-114f-452c-beba-80a7136307ee)

**Comment by unudelaradio** [comment]
Issues with 6443 port and non audio seems to be solved with the Rolling Release #[f0a9d37] (2025-01-02 20:42).

The only issue that still remains is the one with reordering tracks in Reorder Playlist popup, for which i should maybe open a new topic.

**Comment by BusterNeece** [comment]
Yeah, this should be resolved at this point. I was trying to future-proof the application a bit, and it had some unintended side effects, so I reverted the relevant changes.
